src/main/java/org/jabref/model/strings/StringUtil.java

```diff
@@ -750,4 +750,15 @@ public static String quoteStringIfSpaceIsContained(String string) {
             return string;
         }
     }
+
+    /**
+     * Checks if the given string contains any whitespace characters. The supported whitespace characters
+     * are the set of characters matched by {@code \s} in regular expressions, which are {@code [ \t\n\x0B\f\r]}.
+     *
+     * @param s The string to check
+     * @return {@code True} if the given string does contain at least one whitespace character, {@code False} otherwise
+     * */
+    public static boolean containsAnyWhitespaceCharacters(String s) {
```

Comment:
This is not necessary You can use String.isBlank() https://docs.oracle.com/en/java/javase/12/docs/api/java.base/java/lang/String.html#isBlank()
It uses Character.isWhitespace under the hood

Reply:
I added it to prevent typing field names that contain whitespace characters .e.g. "file url". Otherwise, the parsing of the BibTeX will fail.

Reply:
Yes, that's fine, just meaning that you don't need to create a new method for this, but instead use String IsBlank method